addons/website_sale_stock/static/tests/tours/website_sale_stock_multilang.js

```diff
@@ -0,0 +1,64 @@
+/** @odoo-module **/
+
+import tour from 'web_tour.tour';
+
+tour.register('website_sale_stock_multilang', {
+    test: true,
+    url: '/fr/shop?search=unavailable',
+},
+[{
+    content: "Do nothing",
+    trigger: "a",
+    run: () => {}, // This is a check.
+}, {
+    content: "Open unavailable product page",
+    trigger: 'a[content="unavailable_product"]',
+}, {
+    content: "Check out of stock message",
+    trigger: '#out_of_stock_message:contains("Hors-stock")',
+    run: () => {}, // This is a check.
+}, {
+    content: "Check notification message",
+    trigger: '#product_stock_notification_message:contains("Soyez averti")',
```

Comment:
This French sentence isn't in the test data. What if tomorrow someone correct the po file with "Soyez prévenu" or anything else? I don't think the runbot will run before automatically updating a translation... :thinking: Might need some confirmation

Reply:
Fine by me. This lgtm, but I was told this PR is for logistic, so I'll let them manage this =] Have a nice day!